src/components/recordNotVisitingNullIsland.js

```diff
@@ -1,16 +1,18 @@
-var filter = require('through2-filter');
-var _ = require('lodash');
+const filter = require('through2-filter');
```

Comment:
I don't see where this stream is being used. Did it sneak in during the commit or am I missing something?